### PR TITLE
feat: empty runtime

### PR DESCRIPTION
**Pull request by weichweich**

### CODE REVIEW

File: nodes/parachain/res/void.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ],
+  "telemetryEndpoints": null,
+  "protocolId": null,
+  "properties": {
+    "ss58Format": 38,
+    "tokenDecimals": 15,
+    "tokenSymbol": "VOID"
+  },
+  "relay_chain": "kusama",
+  "para_id": 2085,
```
Comment (wischli) [inline nodes/parachain/res/void.json]: Did we reserve this number or this is C&P result?

File: nodes/parachain/Cargo.toml [hunk 2]
```diff
@@ -19,6 +19,7 @@ substrate-build-script-utils = {git = "https://github.com/paritytech/substrate",
 peregrine-runtime = {path = "../../runtimes/peregrine"}
 runtime-common = {path = "../../runtimes/common"}
 spiritnet-runtime = {path = "../../runtimes/spiritnet"}
+empty-runtime = {path = "../../runtimes/empty"}
```
Comment (wischli) [inline nodes/parachain/Cargo.toml:22]: Shouldn't we stick to `void` here to keep it consistent? Else name the chain spec `empty`.

File: runtimes/empty/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+
+impl cumulus_pallet_aura_ext::Config for Runtime {}
+
+parameter_types! {
+	pub const MaxAuthorities: u32 = constants::staking::MAX_CANDIDATES;
+}
+
+impl pallet_aura::Config for Runtime {
+	type AuthorityId = AuthorityId;
+	//TODO: handle disabled validators
```
Comment (wischli) [inline runtimes/empty/src/lib.rs:207]: Can be removed

File: nodes/parachain/Cargo.toml (hunk 2 above)
Comment (weichweich) [reply nodes/parachain/Cargo.toml:22]: yeah noticed halfway through that void is cooler

**Review by wischli — APPROVED** [review]
LGTM. I would question whether we actually want to merge this though? It will create conflicts when merging master into develop after #321.

**Comment by weichweich** [comment]
The fun is over... until the void is called again from the ashes.

**Pull request closed without merging**